README.md

```diff
@@ -141,6 +149,8 @@ Released versions
 -----------------
 See the tagged releases for the following versions of the product:
 
+-v2.6 - JBoss BPM Suite 6.3.0, JBoss EAP 6.4.7 with generic loan demo installed and RH CDK on OSE Cloud install option. 
```

Comment:
There is a missing space here which causes the bullet-list to be correctly generated.